Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/ShellHandRayPointer.cs

```diff
@@ -6,16 +6,21 @@
 
 namespace Microsoft.MixedReality.Toolkit.Input
 {
+    /// <summary>
+    /// Implementation for default hand ray pointers shipped with MRTK. Primariliy used with hands and motion controllers
+    /// </summary>
     [AddComponentMenu("Scripts/MRTK/SDK/ShellHandRayPointer")]
     public class ShellHandRayPointer : LinePointer
     {
+        [Header("Shell Pointer Settings")]
+
         [SerializeField]
         [Tooltip("Used when a focus target exists, or when select is pressed")]
-        private BaseMixedRealityLineRenderer lineRendererSelected = null;
+        private Material lineMaterialSelected = null;
 
         [SerializeField]
-        [Tooltip("Used when no focus target exists and select is not pressed")]
-        private BaseMixedRealityLineRenderer lineRendererNoTarget = null;
+        [Tooltip("Used when a no focus target exists and select is not pressed")]
```

Comment:
I think this was correct before:

```suggestion
        [Tooltip("Used when no focus target exists and select is not pressed")]
```

Reply:
Or something like:

```suggestion
        [Tooltip("Used when a focus target doesn't exist and select is not pressed")]
```